lib/appsignal/probes/mri.rb

```diff
@@ -26,16 +26,13 @@ def call
         @appsignal.set_gauge("gc_runs", GC.count)
 
         gc_stats = GC.stat
+        @appsignal.set_gauge("total_allocated_objects", gc_stats[:total_allocated_objects])
 
-        {
-          :total_allocated_objects => gc_stats[:total_allocated_objects],
-          :major_gc_count => gc_stats[:major_gc_count],
-          :minor_gc_count => gc_stats[:minor_gc_count],
-          :heap_live => gc_stats[:heap_live_slots],
-          :heap_free => gc_stats[:heap_free_slots]
-        }.each do |metric, value|
-          @appsignal.add_distribution_value("gc_stats", value, :metric => metric)
-        end
+        @appsignal.add_distribution_value("gc_count", gc_stats[:minor_gc_count], :metric => :minor_gc_count)
+        @appsignal.add_distribution_value("gc_count", gc_stats[:major_gc_count], :metric => :major_gc_count)
+
+        @appsignal.add_distribution_value("heap_slots", gc_stats[:heap_live_slots], :metric => :heap_live)
+        @appsignal.add_distribution_value("heap_slots", gc_stats[:heap_free_slots], :metric => :heap_free)
```

Comment:
This is just about naming, so feel free to ignore. I'm also not sure if there's a convention that we usually follow for the metric names.

It seems to me that, for the `heap_slots` distribution value, the metric could just be called `live` or `free`, without the `heap_` prefix. Similarly for the `_gc_count` suffix for the metrics for the `gc_count` distribution value.

Reply:
🤷 Up to you! What you said does seem to me like a good reason to keep them this way, so that they're familiar to users who know Ruby GC stuff and what the values for these keys mean.